components/HeaderText.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+} from 'react-native';
+
+export default function HeaderText(props) {
```

Comment:
@msrijai Since the only property you want from props is `title` you can destructure the `props` object to read like this so the code is more concise

```JavaScript
export default function HeaderText({title}) {
    return (
        <Text style={styles.headerText}>{title}</Text>
    )
}
```